node/src/components/block_proposer.rs

```diff
@@ -511,6 +460,12 @@ mod tests {
         }
     }
 
+    impl From<StateStoreRequest> for Event {
+        fn from(_: StateStoreRequest) -> Self {
+            todo!()
```

Comment:
Did you mean to leave this in?

Reply:
Its been properly changed to `unreachable!`